bootstrap/kubeadm/config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml

```diff
@@ -536,8 +536,8 @@ spec:
                               format: date-time
                               type: string
                             value:
-                              description: Required. The taint value corresponding
-                                to the taint key.
+                              description: The taint value corresponding to the taint
+                                key.
```

Comment:
This and other CRD yaml updates are related to the referenced api type description updates in k8s v1.18